apps/parsley/src/components/Search/SearchBar/SearchBar.test.tsx

```diff
@@ -13,7 +13,10 @@ describe("searchbar", () => {
       "aria-disabled",
       "true",
     );
-    expect(screen.getByDataCy("searchbar-input")).toBeDisabled();
+    expect(screen.getByDataCy("searchbar-input")).toHaveAttribute(
```

Comment:
Not sure if you saw this but Leafygreen now recommends using their test harnesses for this. https://mongodb.slack.com/archives/CFFA5BS79/p1712323207260789